ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/WorkloadCvesImageSinglePage.tsx

```diff
@@ -107,17 +107,15 @@ export type ImageDetailsResponse = {
                 digest: string;
             } | null;
         } | null;
-
-        scan: {
-            dataSource: { name: string };
-            scanTime: Date | null;
-        };
+        dataSource: { name: string } | null;
```

Comment:
Note: un-nesting these fields to avoid extra work on the backend as mention by Mandar.